darwin/importer/importer.py

```diff
@@ -1419,7 +1419,8 @@ def process_local_file(local_file):
         files_to_not_track = [
             file_to_track
             for file_to_track in parsed_files
-            if not file_to_track.annotations and (not delete_for_empty)
+            if not (file_to_track.annotations or file_to_track.item_properties)
+            and (not delete_for_empty)
```

Comment:
This is equivalent to:
`not file_to_track.annotations and not file_to_track.item_properties`

Maybe I'm misunderstanding, but shouldn't we want
`not file_to_track.annotations or not file_to_track.item_properties`

Reply:
hmmm think i was misunderstanding and this is correct that we do want to add to `files_not_track` only if `not file_to_track.annotations and not file_to_track.item_properties`